packages/sdk/src/swap/sdk.ts

```diff
@@ -499,7 +499,7 @@ export class SwapSDK {
       dcaParams: quote.type === 'DCA' ? quote.dcaParams : undefined,
       extraParams,
       brokerAccount,
-      brokerCommissionBps,
+      commissionBps: brokerCommissionBps,
```

Comment:
send `commissionBps` instead of `brokerCommissionBps` to backend (i did not want to rename in the zod schema because we use it in other places)